worlds/stardew_valley/strings/seed_names.py

```diff
@@ -66,7 +66,7 @@ class SVESeed:
     slime_seed = "Slime Seed"
     void_seed = "Void Seed"
     shrub_seed = "Shrub Seed"
-    ancient_ferns_seed = "Ancient Ferns Seed"
+    ancient_fern_seed = "Ancient Fern Seed"

```

Comment:
why do all these SVE Seed names have the prefix `_seed` in the constant name?
This makes the usage always look like `SVESeed.shrub_seed`, which feels redundant to me. If it's an SVESeed, we know it's a seed, so it would be fine to do `SVESeed.shrub`